### PR TITLE
RAIL-3609 Cypress page fragments to fluid API

### PR DESCRIPTION
**Pull request by plavemar**
JIRA: RAIL-3609

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by plavemar** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4861/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/10/ : SUCCESS in 12m 27s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4860/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6509/ : SUCCESS in 20m 59s

**Comment by plavemar** [comment]
extended check cypress

File: libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboard.spec.ts [hunk 1]
```diff
@@ -23,44 +23,41 @@ describe("Dashboard", () => {
...
             const topBar = new TopBar();
 
-            topBar.getDashboardTitleElement().should("exist");
-            topBar.getDashboardTitleElement().should("have.text", "E2E RAIL Tests");
+            topBar.dashboardTitleExist();
+            topBar.dashboardTitleHasValue("E2E RAIL Tests");
         });
 
-        it("should not render edit button", () => {
+        it.skip("should not render edit button", () => {
```
Comment (no23reason) [inline libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboard.spec.ts:39]: Are the two skipped tests intentional?

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check-extended-cypress aborted.

**Comment by plavemar** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4866/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/12/ : SUCCESS in 7m 02s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by no23reason** [comment]
extended check cypress

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4868/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/13/ : SUCCESS in 9m 47s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4867/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6514/ : SUCCESS in 16m 22s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1646

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1380/ : SUCCESS in 1m 07s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1656/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/440/ : SUCCESS in 12m 45s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6516/ : SUCCESS in 14m 51s

**Pull request merged**